schemas/theme/translations_schema.json

```diff
@@ -0,0 +1,57 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "description": "A schema for Shopify theme translation files. Supports nested properties, pluralized translations, and translations containing HTML.",
+  "type": "object",
+  "additionalProperties": {
+    "oneOf": [
+      {
+        "type": "string"
```

Comment:
What do you think about adding a description on these items?

I've tried adding a description in this type and I get this doc:

![image](https://github.com/Shopify/theme-liquid-docs/assets/1079279/89fc80cd-2353-4f32-afe8-0c63ee33d6c5)